### ISSUE TITLE
Database and meta information

### ISSUE THREAD

**Issue by lpatiny** [cheminfo/nmrium#1883]
From preferences we should be able to add unlimited number of columns by selecting jpath.

This is related to:

https://github.com/zakodium-oss/analysis-ui-components/issues/252

**Comment by hamed-musallam** [comment]
this is something we need to discuss further because this has complicated because of how the database table is generated.

as you know the database table rows are the signals, The database object tree (spectra>ranges>signals>jcoulings)  and what has left after that we join values.

without reshaping the data we can not have the current shape of the table

![image](https://user-images.githubusercontent.com/35760236/200841928-6e092925-1d59-4ce4-9420-07ccd2e76c5f.png)

**Comment by lpatiny** [comment]
# counter / selection

Should be 26 / 30

![Image](https://github.com/user-attachments/assets/54329381-b92e-4cad-9621-90ffd21d1fb0)

# Minimal row height

We should setup a minimal size for a row. 200px could be reasonable. This should be a module preference.

# When adding spectrum, switch to stack

**Comment by hamed-musallam** [comment]
@lpatiny 
> We should setup a minimal size for a row. 200px could be reasonable. This should be a module preference.

Do you mean making it available in the database preferences? it is not a row preference, it’s more of a choice related to the molecule structure column.

![Image](https://github.com/user-attachments/assets/7811136c-758b-43c9-a288-260bc79d5da1)

**Comment by lpatiny** [comment]
You are right it is indeed for the molecule and yes in the database preferences.
